src/core/doall/include/arcana/gino/core/DOALL.hpp

```diff
@@ -57,18 +57,22 @@ class DOALL : public ParallelizationTechnique {
   Noelle &n;
   std::map<PHINode *, std::set<Instruction *>> IVValueJustBeforeEnteringBody;
 
-  virtual void invokeParallelizedLoop(LoopContent *LDI);
+  virtual void invokeParallelizedLoop(LoopContent *LDI,
+                                      bool loopHasParallelizedOutput);
 
   /*
    * DOALL specific generation
    */
   void rewireLoopToIterateChunks(LoopContent *LDI, DOALLTask *task);
+  bool replaceOutputSequences(LoopContent *LDI);
```

Comment:
I'm not sure why "Sequences"

Reply:
Potentially there could be different kinds. Mostly left over from when I thought different output destinations would be different SCCs. Changed.